### DISCUSSION TITLE
Material model

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#256] [Q&A]
I wonder in the Peridigm material model, what's the difference between "Elastic" and "Elastic Correspondence"? Or the difference between "Elastic Plastic" and "Elastic Plastic Correspondence". Thank you.

**Comment by ppandit95** [comment]
Hi @guangdongzhang 

the "Elastic" model simulates the elastic behaviour with ordinary state based peridynamics while "Elastic Correspondence" model simulates the elastic behaviour with non - ordinary state based peridynamics.Both these models work well for isotropic material but when we have a complex material morphology then correspondence model gives better results .

The "Elast Plastic" model simulates elastic-perfectly plastic behaviour with ordinary state based peridynamics while "Elastic Plastic Correspondence" does so with non ordinary state based peridynamics.

Hope this helps.

Regards
Pushkar
